# LDC LUT definition file path
lut_file_path:          $(find zed_capture)/config/SN5867575_HD_LUT_right.bin

# location of DL model
dl_model_path:    $(env TI_DATA_PATH)/tidl_semseg_model/tvmdlr_deeplabv3lite_mobilenetv2_tv_768x432_qat-p2_onnx
# dl_model_path:    $(env TI_DATA_PATH)/tidl_semseg_model/onnxrt_deeplabv3lite_mobilenetv2_tv_768x432_qat-p2_onnx

## Models below are all segmentation models from edge AI model zoo. Not all models yet tested.
# dl_model_path: /opt/model_zoo/ONR-SS-8610-deeplabv3lite-mobv2-ade20k32-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8630-unetlite-aspp-mobv2-tv-ade20k32-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8650-fpnlite-aspp-mobv2-ade20k32-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8670-fpnlite-aspp-mobv2-1p4-ade20k32-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8690-fpnlite-aspp-regnetx400mf-ade20k32-384x384
# dl_model_path: /opt/model_zoo/ONR-SS-8700-fpnlite-aspp-regnetx800mf-ade20k32-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8710-deeplabv3lite-mobv2-cocoseg21-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8720-deeplabv3lite-regnetx800mf-cocoseg21-512x512
# dl_model_path: /opt/model_zoo/ONR-SS-8730-deeplabv3-mobv3-lite-large-cocoseg21-512x512
# dl_model_path: /opt/model_zoo/TFL-SS-2580-deeplabv3_mobv2-ade20k32-mlperf-512x512
# dl_model_path: /opt/model_zoo/TVM-SS-5720-deeplabv3lite-regnetx800mf-cocoseg21-512x512

# input data width in pixels
width:                  1280

# input data height in lines
height:                 720

# Maximum number of Semantic Segmentation classes
num_classes:            20

# interactive mode
is_interactive:         0

# to enable showing output on display when display_option = 1
display_option:         1

# pipeline depth
pipeline_depth:         4

# flag to indicate if the graph should be exported
#  0 - disable
#  1 - enable
exportGraph:            0

# real-time logging enable.
#  0 - disable
#  1 - enable
rtLogEnable:            0

# log level for the app
# 0 - DEBUG: Used to show general debug messages
# 1 - INFO: Used to show run-time processing debug
# 2 - WARN: Used to warning developers of possible issues
# 3 - ERROR: Used for most errors
log_level:              2
